recipes/earthpy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "earthpy" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f604ffe3666d59f4104bff2ca887312c4c7b827a87f8716561ae86b5e9b76a05
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - geopandas
+    - matplotlib >=2
+    - numpy
+    - pandas
+    - rasterio
+    - Rtree>=0.8
+    - scikit-image
+
+
+test:
+  imports:
+    - earthpy
+    - earthpy.clip
+    - earthpy.io
+    - earthpy.mask
+    - earthpy.plot
+    - earthpy.spatial
+
+about:
+  home: http://github.com/earthlab/earthpy
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Utility functions for the working with spatial data'
+
+  description: |
+    A set of helper functions to make working with spatial data in open source
+    tools easier. This package is maintained by Earth Lab and was originally
+    designed to support the earth analytics education program.
+  doc_url: http://earthpy.readthedocs.io/
+  dev_url: https://github.com/earthlab/earthpy
+
+extra:
+  recipe-maintainers:
+    - lwasser
```

Comment:
@lwasser Are you happy to be listed as a maintainer of this recipe?

Reply:
@chrisburr  i am!! thanks